web/assets/js/components/dashboard.ts

```diff
@@ -52,14 +67,6 @@ const Dashboard = class extends Component {
     return rating;
   }
 
-  async _feedUserBlitzRating() {
```

Comment:
@zivkovicmilos You have removed these feed methods, but can we finally get it by category or should I remove / update all the dashboard UI? If we cannot have the data, I should change all the UI...

Reply:
Same question for the leaderboard @zivkovicmilos @thehowl : Can we have `getLeaderboard` by categories and player score by category?

Reply:
If not possible to get the distinction the user, I will transform the dashboard UI like this. I've remove the 2 category and use one tile as absolute game perf, another as score/position display and the last one a relative (%) perf.

<img width="1087" alt="Capture d’écran 2023-09-20 à 00 52 35" src="https://github.com/gnolang/gnochess/assets/7622257/879ae841-915d-4654-a781-aaaabf8dda37">

Reply:
See https://github.com/gnolang/gnochess/pull/30

Reply:
@alexiscolin 
As far as I can see, there is no fine way to get the [leaderboard by category](https://github.com/gnolang/gnochess/blob/57d647373a6867a3a9d59bce01ed83635f8dbbad/realm/discovery.gno), only the current leaderboard:
https://github.com/gnolang/gnochess/blob/57d647373a6867a3a9d59bce01ed83635f8dbbad/realm/discovery.gno#L201-L218

Reply:
Do you confirm @thehowl ? How will we get the champion by category if this method does not exist? 
i'm gonna start updating the leaderboard in // of our tests anyway :)